lib/kitchen/patches/nokogiri.rb

```diff
@@ -71,7 +71,15 @@ def previous
         prev = previous_element
```

Comment:
So this is actually a different method than previous/previous_sibling that only returns the previous element. We might not need the updated previous method or text checks at all.